reference/metrics.html.md

```diff
@@ -31,28 +31,6 @@ Instrument your app and expose your metrics on `0.0.0.0`.
 
 There are many supported [client libraries](https://prometheus.io/docs/instrumenting/clientlibs/) as well as 3rd party libraries able to return Prometheus-formatted metrics.
 
-## Sending custom metrics to Prometheus
-
-Apps can expose a metrics endpoint we'll periodically (a few times per minute).
-
-### Configuration
-
-Add this to your application's `fly.toml`
-
-```toml
-app = "your-app-name"
-
-[experimental]
-metrics_port = 9091 # default for most prometheus clients
-metrics_path = "/metrics" # default for most prometheus clients
-```
-
-### Instrumentation
```

Comment:
We actually want to keep this section, it's important!

Reply:
Are you referring to the instrumentation section? If so, it's already present on lines 28-32.

Reply:
Oh yes, I misunderstood the diff. All good!